tests/src/multi-store/multi-store.module.ts

```diff
@@ -1,24 +1,18 @@
 import { Module } from '@nestjs/common';
-import { redisStore } from 'cache-manager-redis-store';
 import { CacheModule } from '../../../lib';
 import { MultiStoreController } from './multi-store.controller';
+import KeyvRedis from '@keyv/redis';
+import { Keyv } from 'keyv';
+import { CacheableMemory } from 'cacheable';
 
 @Module({
   imports: [
-    CacheModule.register([
-      {
-        store: 'memory',
-        max: 100,
-        ttl: 50,
-      },
-      {
-        store: redisStore,
-        host: 'localhost',
-        port: 6379,
-        db: 0,
-        ttl: 50,
-      },
-    ]),
+    CacheModule.register({
+      stores: [
+        new Keyv({ store: new CacheableMemory({ ttl: 60000, lruSize: 5000 }) }),
+        new KeyvRedis('redis://192.168.100.1:6379'),
```

Comment:
You are changing the default from `localhost` to `192.168.100.1` why?

Reply:
maybe impl keyv adapter
```
implements KeyvStoreAdapter
```
i think this package only use cachemanager with nestjs, now cachemanager is supported by keyv, use keyv adapter can better maintain consistency。


test case redis ip use 192.168.100.1,because my machine not install Redis, there is indeed a lack of consideration here